backend/test/institution_handlers_tests/institution_handler_test.py

```diff
@@ -6,8 +6,8 @@
 from models import User
 from models import Institution
 from handlers.institution_handler import InstitutionHandler
-from worker import AddAdminPermissionsInInstitutionHierarchy
-from worker import RemoveAdminPermissionsInInstitutionHierarchy
+from worker_handler import AddAdminPermissionsInInstitutionHierarchyHandler
+from worker_handler import RemoveAdminPermissionsInInstitutionHierarchyHandler
```

Comment:
Simplify those handler's names. I think you can remove the 'InInstitutionHierarchy' part.